README.md

```diff
@@ -12,70 +12,23 @@ PySTAC is a library for working with [SpatioTemporal Asset Catalog](https://stac
 
 ## Installation
 
-PySTAC requires Python >= 3.8. This project follows the recommendations of
-[NEP-29](https://numpy.org/neps/nep-0029-deprecation_policy.html) in deprecating support
-for Python versions. This means that users can expect support for Python 3.8 to be
-removed from the `main` branch after Apr 14, 2023 and therefore from the next release
-after that date.
-
-PySTAC has a single required dependency (`python-dateutil`).
-PySTAC can be installed from pip or the source repository.
+### Install from PyPi (recommended)
 
 ```shell
 pip install pystac
 ```
 
-If you would like to enable the validation feature utilizing the
-[jsonschema](https://pypi.org/project/jsonschema/) project, install with the optional
-`validation` requirements:
-
-```shell
-pip install pystac[validation]
-```
-
-If you would like to use the [`orjson`](https://pypi.org/project/orjson/) instead of the
-standard `json` library for JSON serialization/deserialization, install with the
-optional `orjson` requirements:
-
-```shell
-pip install pystac[orjson]
-```
-
-From source repository:
-
-```shell
-git clone https://github.com/stac-utils/pystac.git
-cd pystac
-pip install .
-```
```

Comment:
I think these sections can stay in the README. The `jsonschema` feature in particular is oft-used and should be easy to find.